### DISCUSSION TITLE
Tap to increment numerical habits

### DISCUSSION THREAD

**Discussion by alxmjo** [iSoron/uhabits#1218] [Feature Requests]
The new numerical tracker is great addition to the app. It would be even better if we had the option to increment the value by tapping, rather than clicking to open a submenu and then inputting the value there. 

This would be especially valuable for habits like "How many glasses of water do I drink a day?" I'm more likely to track that over the course of the day rather than keep a running sum in my head, so the ability to tap to increment over the course of the day would be very helpful. 

I think the tap to increment function would be best in the widget, but would also be nice from inside the app itself.

Edit: I'd say that this is not a duplicate of #951 since what I'm advocating here is to skip the popup entirely and just increment from a single tap.

**Comment by hiqua** [comment]
Dup: #951
